types/reactjs-localstorage/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for reactjs-localstorage 1.0
+// Project: https://github.com/null-none/react-localstorage
+// Definitions by: rwilson504 <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/reactjs-localstorage/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me